### ISSUE TITLE
[Rendering] Teams 2.1/New Channels Experience "full width" cards not so full width. 

### ISSUE THREAD

**Issue by sayali-MSFT** [MicrosoftDocs/msteams-docs#9602]
Copied Issue Link-https://github.com/microsoft/AdaptiveCards/issues/8692

**Target Platforms**
UWP, Other

**SDK Version**
Unsure

**Application Name**
Microsoft Teams 2.1/New Channels Experience

**Problem Description**

When using
"msteams": { "width": "full" }
in an adaptive card, the classic Teams client will render it properly as the full width of the conversation box:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93527375/d12424de-e757-45d3-b927-171db0581c98)

However, in Teams 2.1 or cases where the New Channel Experience has become the default in Teams Classic, the adaptive card width is no more than around two thirds of the conversation box:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93527375/f698f739-cb92-4bfa-97c6-b77f4411c03b)

Screenshots
No response

```
Card JSON
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.4",
    "msteams": {
        "width": "Full"
    },
    "body": [
        {
            "type": "Container",
            "bleed": true,
            "style": "emphasis",
            "items": [
                {
                    "type": "ColumnSet",
                    "columns": [
                        {
                            "type": "Column",
                            "width": "auto",
                            "items": [
                                {
                                    "type": "Image",
                                    "altText": "Announcement",
                                    "url": "https://normanhurb-h.schools.nsw.gov.au/content/dam/doe/sws/schools/n/normanhurb-h/icons/Communication.png",
                                    "width": "75px",
                                    "selectAction": {
                                        "type": "Action.OpenUrl",
                                        "url": "https://schoolsnsw.sharepoint.com/sites/N1XHSGBP/SitePages/Modern-Home.aspx#%F0%9F%93%A2-announcements",
                                        "id": "11",
                                        "title": ""
                                    }
                                }
                            ]
                        },
                        {
                            "type": "Column",
                            "width": "stretch",
                            "items": [
                                {
                                    "type": "TextBlock",
                                    "text": "**Whole School Announcement Summaries** ",
                                    "weight": "Bolder",
                                    "size": "Large",
                                    "wrap": true,
                                    "color": "Attention"
                                }
                            ],
                            "verticalContentAlignment": "Center"
                        }
                    ]
                }
            ]
        },
        {
            "type": "TextBlock",
            "text": "This thread contains announcement summaries that are posted at \n- 8:00 am\n- 12:00 pm (midday)\n- 4:00 pm\n\neach day, where new announcement have been made.",
            "wrap": true,
            "separator": true,
            "size": "Small",
            "color": "Accent"
        },
        {
            "type": "ActionSet",
            "actions": [
                {
                    "type": "Action.OpenUrl",
                    "title": "All Whole School Announcements",
                    "url": "https://schoolsnsw.sharepoint.com/sites/N1XHSGBP/SitePages/Modern-Home.aspx#%F0%9F%93%A2-announcements",
                    "style": "positive",
                    "id": "2",
                    "iconUrl": "https://normanhurb-h.schools.nsw.gov.au/content/dam/doe/sws/schools/n/normanhurb-h/icons/sharepoint.png"
                },
                {
                    "type": "Action.OpenUrl",
                    "title": "Manage announcements",
                    "url": "https://schoolsnsw.sharepoint.com/sites/N1XHSGBP/Lists/Whole%20School%20Announcements/manage-announcement.aspx",
                    "style": "positive",
                    "id": "3",
                    "iconUrl": "https://normanhurb-h.schools.nsw.gov.au/content/dam/doe/sws/schools/n/normanhurb-h/icons/sharepoint.png"
                }
            ]
        }
    ]
}
```
Sample Code Language
No response

Sample Code
No response

**Comment by z3019494** [comment]
Hi all, the issue seems to have been resolved on Teams 2.1, but not Teams 1.0 on the web.

https://github.com/microsoft/AdaptiveCards/issues/8692#issuecomment-1746179594

**Comment by sayali-MSFT** [comment]
@z3019494 -We have tested this scenario on Teams 1.0 on the web. It is displaying the full-width adaptive card properly on desktop/Web and we can confirm that as well.
Desktop- Microsoft Teams Version 1.6.00.27567 (64-bit)

![full-widthAD](https://github.com/MicrosoftDocs/msteams-docs/assets/93527375/b71b48f7-9529-47c2-9ff4-f30cff59b050)

**Comment by z3019494** [comment]
> @z3019494 -We have tested this scenario on Teams 1.0 on the web. It is displaying the full-width adaptive card properly on desktop/Web and we can confirm that as well. Desktop- Microsoft Teams Version 1.6.00.27567 (64-bit)
> 
> ![full-widthAD](https://user-images.githubusercontent.com/93527375/273223739-b71b48f7-9529-47c2-9ff4-f30cff59b050.png)

Hi @sayali-MSFT , it seems to be fixed on Teams 1.0 on the web as well using the New Channels Experience:

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/60587625/ab591296-3457-4630-8623-a581302428b5)

**Comment by sayali-MSFT** [comment]
@z3019494 - We are closing this issue for now, please feel free to reopen it, if required.

Could you please share your valuable feedback via [Microsoft Teams Developer Feedback link](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR7iCOpS5_b9Nqmwx43u5rtZUOThVR081SllSR05aSDQxQ0tUMDVPTVIxTi4u).
